applications/serial_lte_modem/Kconfig

```diff
@@ -25,6 +25,9 @@ config SLM_AT_MAX_PARAM
 config SLM_NATIVE_TLS
 	bool "Use Zephyr mbedTLS"
 
+config SLM_UART_HWFC_RUNTIME
```

Comment:
No, unnecessary CONFIG, should be always supported

Reply:
Config removed.